akka-sse/src/test/scala/de/heikoseeberger/akkasse/ServerSentEventParserSpec.scala

```diff
@@ -27,20 +27,21 @@ class ServerSentEventParserSpec extends BaseSpec {
   "A ServerSentEventParser" should {
 
     "parse ServerSentEvents correctly" in {
-      val input = """|data: message 1 line 1
-                     |data:message 1 line 2
-                     |
-                     |data: message 2
-                     |:This is a comment and must be ignored
-                     |event: Only the last event should be considered
-                     |event: message 2 event
-                     |
-                     |data:
-                     |
-                     |event: message 4 event
-                     |
-                     |data: incomplete message
-                     |""".stripMargin
+      val input = List(
+        "data: message 1 line 1",
+        "data:message 1 line 2",
+        "",
+        "data: message 2",
+        ":This is a comment and must be ignored",
+        "event: Only the last event should be considered",
+        "event: message 2 event",
+        "",
+        "data:",
+        "",
+        "event: message 4 event",
+        "",
+        "data: incomplete message",
+        "").mkString("\n")
```

Comment:
Could you please retry on Windows with `f"%n"` instead of `"\n"`, which translates to the native line separator?